### DISCUSSION TITLE
Failed to list - Broken Pipe Errors

### DISCUSSION THREAD

**Discussion by kevinmatthews-kpmg** [kedacore/keda#1406] [General]
Hi,

I was alerted to an error with our keda scaling this morning, when I looked at the logs I found the following:

```
E1208 09:09:53.190858       1 reflector.go:126] pkg/mod/k8s.io/client-go@v11.0.1-0.20190409021438-1a26190bd76a+incompatible/tools/cache/reflector.go:94: Failed to list *v1alpha1.TriggerAuthentication: Get "https://10.0.0.1:443/apis/keda.k8s.io/v1alpha1/triggerauthentications?limit=500&resourceVersion=0": write tcp 10.244.0.98:51748->10.0.0.1:443: write: broken pipe
E1208 09:09:53.191955       1 reflector.go:126] pkg/mod/k8s.io/client-go@v11.0.1-0.20190409021438-1a26190bd76a+incompatible/tools/cache/reflector.go:94: Failed to list *unstructured.Unstructured: Get "https://10.0.0.1:443/apis/keda.k8s.io/v1alpha1/namespaces/keda/scaledobjects?limit=500&resourceVersion=0": write tcp 10.244.0.98:51748->10.0.0.1:443: write: broken pipe
```

There was a LOT of these over the course of a few hours... a restart of keda operator deployment seemed to do the job, but can anyone shed light on to what this issue relates to in order to avoid this issue going forward?

Thanks
Kevin

**Comment by zroubalik** [comment]
Thanks for the report Kevin, seems like the underlying client-go has lost it's connection, it might be some random issue in the communication with the k8s apiserver. I am afraid that there's probably nothing we could do currently :(  

Please let us know, if you happen to find this kind of error again!
